lib/beaker-puppet/install_utils/foss_utils.rb

```diff
@@ -602,7 +602,7 @@ def install_a_puppet_msi_on(hosts, opts)
 
             if host.is_cygwin?
               # NOTE: it is critical that -o be before -O on Windows
-              on host, "curl -o \"#{msi_download_path}\" -O #{link}"
+              on host, "export http_proxy=#{opts[:package_proxy]}  && curl -o \"#{msi_download_path}\" -O #{link}"
```

Comment:
I believe there is no support for a windows environment to use a proxy like for instance the opentable vagrant boxes. Are there any plans to add support for a proxy?